CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ option(ALSOFT_INSTALL_EXAMPLES "Install example programs (alplay, alstream, ...)
 option(ALSOFT_INSTALL_UTILS "Install utility programs (openal-info, alsoft-config, ...)" ON)
 option(ALSOFT_UPDATE_BUILD_VERSION "Update git build version info" ON)
 
+option(ALSOFT_OSX_FRAMEWORK "Build as macOS framework support" ON)
```

Comment:
Could you please disable this by default?

Reply:
Yeah, should disable by default